packages/nuxt/src/core/plugins/plugin-metadata.ts

```
@@ -47,36 +46,31 @@ export async function extractMetadata (code: string, loader = 'ts' as 'ts' | 'ts
+    }
 
-      meta.order = meta.order || orderMap[meta.enforce || 'default'] || orderMap.default
-      delete meta.enforce
-    },
+    const plugin = node.arguments[0]
+    if (plugin?.type === 'ObjectExpression') {
+      meta = defu(extractMetaFromObject(plugin.properties), meta)
+    }
+
+    meta.order = meta.order || orderMap[meta.enforce || 'default'] || orderMap.default
+    delete meta.enforce
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)